### DISCUSSION TITLE
spack environment

### DISCUSSION THREAD

**Discussion by auraoupa** [nextsimdg/nextsimdg#705] [General]
Hi @TomMelt, I tried to implement the spack environment with the spack.yml file on two different machines (linux and mac) and I get the same error each time : ```Cannot satisfy 'cmake@3.27.7'``` and even when I remove the exact version the environment building does not go through ... 
On which device did you set up this environment ?

**Comment by TomMelt** [comment]
hmmm... interesting. I have not had this issue before. I will try running it in a vm and see what happens

**Reply by TomMelt** [reply]
I used an ubuntu 24.04 VM and I indeed found the same thing. It is due to recent deprecation of `cmake@3.27.7` by `spack`. I have made a new PR #707 that fixes this (at least for my VM). Please let me know if it works for you too :crossed_fingers:
